cirq-core/cirq/study/result_test.py

```diff
@@ -166,6 +168,21 @@ def test_df():
     assert df.c.value_counts().to_dict() == {0: 3, 1: 2}
 
 
+def test_df_large():
+    result = cirq.ResultDict(
+        params=cirq.ParamResolver({}),
+        measurements={
+            'a': np.array([[0 for _ in range(76)]] * 10_000, dtype=bool),
+            'd': np.array([[1 for _ in range(76)]] * 10_000, dtype=bool),
+        },
+    )
+
+    assert np.all(result.data['a'] == 0)
+    assert np.all(result.data['d'] == 75557863725914323419135)
```

Comment:
Can we add a comment or something to explain this magic number?

Reply:
Would perhaps be clearer if written in hex (underscores are optional, just for grouping):
```suggestion
    assert np.all(result.data['d'] == 0xfff_ffffffff_ffffffff)
```

Reply:
ah you stole my suggestion